sys/mips/uart_cbus.c

```diff
@@ -28,6 +31,12 @@ static bool is_set(unsigned offset, uint8_t mask) {
 }
 
 static void cbus_uart_init(console_t *dev __unused) {
+  vaddr_t handle = kva_alloc(PAGESIZE);
+  /* MALTA_CBUS_UART is MALTA_FPGA_BASE + 0x900 */
```

Comment:
Maybe `MALTA_CBUS_UART` should be set as relative to `MALTA_FPGA_BASE` to avoid 0x900 below?

Reply:
done